plugins/CoreHome/templates/_topBar.twig

```diff
@@ -3,7 +3,7 @@
 
     {% macro menuItemLabel(label, icon) %}
         {% if icon is defined and icon and icon starts with 'icon-' %}
-            <span class="{{ icon|striptags }}"></span>
+            <span class="{{ icon|striptags }}" aria-label="{{ label|translate }}"></span>
```

Comment:
Do you mind changing it to `{{ label|translate|e('html_attr) }}` for better escaping? Cheers